Makefile

```diff
@@ -35,6 +45,14 @@ lint-scripts: ## Validate shell scripts
 		-w /src \
 		${DOCKER_IMAGE_SHELLCHECK} \
 		$(VERIFY_SCRIPTS)
+.PHONY: release
+release: ## Release Changed Helm Charts
+	DOCKER_IMAGE_CT="$(DOCKER_IMAGE_CT)" \
+	DOCKER_IMAGE_CR="${DOCKER_IMAGE_CR}" \
+	GIT_USER="${GIT_USER}" \
+	GIT_EMAIL="${GIT_EMAIL}" \
+	 	scripts/release.sh
+
```

Comment:
Would propose this part of code, to be performed in a Github Action. 
The release process, is not something that will be run in development phase. 

Only the build/release system will have to run it. 

It will also, make the release.sh script obsolete. 
No need to bugfix/maintain it.

Reply:
if there is a bug in release build how we can reproduce what CI did? we need to go to actions source code and understand what command it uses right?

Reply:
Having that script helped alot during development to test locally. Also it will help alot to fix the future problems. If we move everything to the Github Actions, it will be very hard for us to make development.

Reply:
I'd challenge that. 
https://github.com/nektos/act

There is also possibility to connected to ghaction instance for debugging reasons. 
https://github.com/marketplace/actions/debugging-with-ssh

From my point of view, less/zero code maintenance, is the better approach here. 
But not strongly opinionated.

Reply:
@spirosoik @pfltdv  
Imaging there is a vurnerability introduced, in a binary that the script runs. 

Q: How will we know about this? 
Q: Do you think there will be enough time from detection, to actual commiting all of our repos that contain the vurenrability?
Q: No standardization, the release code commited in the makefile, can be whatever.

Reply:
@phoinixgrr Thanks alot for sharing details about act. Looks very promising tool. Is it possible to discuss to use act and makefile in next DevOps meeting and apply that decision into our Unified CI document as a policy. And change the pipeline at follow-up iteration if we decide to use that tool. Since the scope of this ticket to fix race condition and drop CircleCI pipelines.

Reply:
This looks really nice. let's add this in the next meeting and may a demo as Mustafa mentioned.